extendr-api/src/wrapper/matrix.rs

```diff
@@ -361,41 +361,60 @@ impl<T, D> Deref for RArray<T, D> {
     }
 }
 
-#[test]
```

Comment:
Maintenance: Test functions should be in their own module.

Reply:
I am reworking this `matrix.rs` entirely. If you could remove this change from your PR (I can create this test module in mine). I will make the PR in one week.